content/docs/aws/aws-e2e.md

```diff
@@ -0,0 +1,362 @@
+ip-192-168-63-25.eu-west-1.compute.internal    Ready    <none>   18d   v1.14.7-eks-1861c5
+ip-192-168-68-104.eu-west-1.compute.internal   Ready    <none>   18d   v1.14.7-eks-1861c5
+ip-192-168-77-56.eu-west-1.compute.internal    Ready    <none>   18d   v1.14.7-eks-1861c5
+```
+
+If you'd like to change the nodegroup scaling there are two options, either via the EC2 auto-scaling group or using `eksctl`: 
+```shell script
+eksctl scale nodegroup --cluster=aiplatform --nodes=4 ng
+```
+
+### Deploy the kubernetes dashboard
+
+To deploy the kubernetes dashboard as described in the [AWS deploy kubernetes web ui](https://docs.aws.amazon.com/eks/latest/userguide/dashboard-tutorial.html), first download and install the metrics server:
+
+To install the metrics server:
+```shell script
+wget https://api.github.com/repos/kubernetes-sigs/metrics-server/tarball/v0.3.6
+tar zxvf v0.3.6
+kubectl apply -f kubernetes-sigs-metrics-server-d1f4f6f/deploy/1.8+
+```
+Validate:
+```shell script
+kubectl get deployment metrics-server -n kube-system
+```
+```
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+metrics-server   1/1     1            1           18d
+```
+
+To install the dashboard and create a user to access it, first create an `eks-admin` user using the following file:
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: eks-admin
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: eks-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: eks-admin
+  namespace: kube-system
+```
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-rc5/aio/deploy/recommended.yaml
+kubectl apply -f eks-admin-service-account.yaml
+```
+
+To access the kubernetes dashboard bring it to your localhost with a proxy:
+
+```shell script
+kubectl proxy
+```
+
+And then visit the dashboard on the [kubernetes dashboard ui](http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/#!/login)
+
+Exposing the kubernetes dashboard via an istio virtual service is not recommended.
+
+To login get the token using the following command:
+```shell script
+kubectl -n kube-system describe secret $(kubectl -n kube-system get secret | grep eks-admin | awk '{print $1}')
+```
+
+More information on [creating and managing EKS clusters](https://eksctl.io/usage/creating-and-managing-clusters/).
+
+## Deploy Kubeflow
+
+In this section you will prepare the ecosystem required by kubeflow, and you will configure the kfctl.yaml file with the custom information for your environment.
+
+### Cognito and certificates
+
+#### Route53
+
+Register or delegate a domain in a Route53 hosted zone, in our example we have delegated the subdomain platform.domain.com.
```

Comment:
I am not sure if this is clear enough for most of the users.

Reply:
addressed in commit 9ffa0d1